cli/cmd/get.go

```diff
@@ -480,14 +489,73 @@ func describeAPI(name string, resourcesRes *schema.GetResourcesResponse) (string
 		sort.Strings(samplePlaceholderFields)
 		samplesPlaceholderStr := `{ "samples": [ { ` + strings.Join(samplePlaceholderFields, ", ") + " } ] }"
 		out += "Payload:  " + samplesPlaceholderStr + "\n"
-	}
-	if api != nil {
-		out += titleStr("Configuration") + api.UserConfigStr()
+	} else {
+		out += titleStr("Model Input Signature")
+
+		if groupStatus.Available() == 0 {
+			out += "Model "
```

Comment:
Is this intended? Maybe this check should happen before `out += titleStr("Model Input Signature")`?